### PR TITLE
Brand tests

### LINKED ISSUE #334: Brand: Add tests

### PR DESCRIPTION
**Pull request by westwood846**
Closes #334

First of all this PR adds a config option to package.json that lets snapshots deal with references to assets like jpgs.

And then I had some trouble in the tests because `Brand` sets `window.location.href` on right-click. Of course this threw an error in the tests but I found out that I can work around that by overwriting `window.location` with my own (empty) object.

### CODE REVIEW

File: src/General/Brand/Brand.test.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import 'jest-styled-components';
+import { render, fireEvent } from '@testing-library/react';
+// import '@testing-library/jest-dom/extend-expect';
```
Comment (ritikamotwani) [inline src/General/Brand/Brand.test.tsx]: Any reason for this comment?

File: src/General/Brand/Brand.test.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const { getByRole } = render(
+      <Brand
+        asset="glints-black"
+        rightClickURL={url}
+        onContextMenu={onContextMenu}
+      />
+    );
+    const container = getByRole('presentation');
+    fireEvent.contextMenu(container);
+    expect(onContextMenu).toHaveBeenCalledTimes(1);
```
Comment (ritikamotwani) [inline src/General/Brand/Brand.test.tsx:51]: Do we have to show that this wasn't called before the fireEvent? :P

File: src/General/Brand/Brand.test.tsx (hunk 1 above)
Comment (westwood846) [reply src/General/Brand/Brand.test.tsx]: Removed

File: src/General/Brand/Brand.test.tsx (hunk 2 above)
Comment (westwood846) [reply src/General/Brand/Brand.test.tsx:51]: Added an expectation to be sure.

**Review by ritikamotwani — APPROVED** [review]

**Pull request merged**